### PR TITLE
GridView - fix after gridListIItem change (#3001)

### PR DESCRIPTION
**Pull request by M-i-k-e-l**
## Description
GridView - fix after gridListIItem change (#3001)

## Changelog
GridView - fix after gridListIItem change (#3001)

## Additional info
None

### CODE REVIEW

File: src/components/gridListItem/index.tsx [hunk 1]
```diff
@@ -213,7 +213,12 @@ class GridListItem extends Component<GridListItemProps> {
         accessible={renderCustomItem ? true : undefined}
         {...Modifiers.extractAccessibilityProps(this.props)}
       >
-        {imageProps && <Image style={itemSize} {...imageProps} customOverlayContent={children}/>}
+        {imageProps && (
+          <View style={[{borderRadius: imageProps?.borderRadius}, itemSize]}>
+            <Image {...imageProps} style={[itemSize, imageProps?.style]}/>
```
Comment (adids1221) [inline src/components/gridListItem/index.tsx:218]: I found that the issue is coming from the style override.
```
style={itemSize} {...imageProps}
```
So when the user pass some style via imageProps the size getting override.

I think the better solution will be to fix the style override.

```
{imageProps && <Image {...imageProps} style={[itemSize, imageProps?.style]} customOverlayContent={children}/>}
```

**Review by adids1221 — COMMENTED** [review]
Left a small comment, the issue coming from style override.

File: src/components/gridListItem/index.tsx (hunk 1 above)
Comment (M-i-k-e-l) [reply src/components/gridListItem/index.tsx:218]: So create a PR, if it works I'll close this one

Comment (adids1221) [reply src/components/gridListItem/index.tsx:218]: [related fix](https://github.com/wix/react-native-ui-lib/pull/3222)

**Pull request closed without merging**